src/Microsoft.OData.Edm/Vocabularies/Annotations/EdmVocabularyAnnotation.cs

```diff
@@ -44,8 +48,213 @@ public EdmVocabularyAnnotation(IEdmVocabularyAnnotatable target, IEdmTerm term,
             this.term = term;
             this.qualifier = qualifier;
             this.value = value;
+            this.usesDefault = false;
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the EdmVocabularyAnnotation class
+        /// to the default value.
+        /// </summary>
+        /// <param name=“target”>Element the annotation applies to.</param>
+        /// <param name=“term”>Term bound by the annotation</param>
+        public EdmVocabularyAnnotation(IEdmVocabularyAnnotatable target, IEdmTerm term)
+        {
+            // Check arguments
+            if (term.DefaultValue == null) // throw error if no default value
+            {
+                // Note: I'm not sure which is the better way of throwing the exception:
+                //throw new ArgumentNullException("value");
+                EdmUtil.CheckArgumentNull(new EdmStringConstant(null), "value");
+            }
+            EdmUtil.CheckArgumentNull(target, "target");
+            EdmUtil.CheckArgumentNull(term, "term");
+
+            // Check arguments
+            EdmUtil.CheckArgumentNull(target, "target");
+            EdmUtil.CheckArgumentNull(term, "term");
+
+            // Initialize
+            this.target = target;
+            this.term = term;
+            this.qualifier = null;
+            //this.value = BuildDefaultValue(term); // Note: Eventually this method will parse the default value
+            this.value = new EdmStringConstant(term.DefaultValue); // Note: temporarily only supporting Strings
+            this.usesDefault = true;
+        }
+
+        /// <summary>
+        /// Parses a <paramref name="defaultValue"/> into an IEdmExpression value of the correct type.

```

Comment:
We might want to move this logic (once it works) to a general helper class so that it can be called from multiple locations (i.e., EdmVocabularyAnnotation as well as CsdlSemanticsVocabularyAnnotation)

Reply:
Done ✅